stories/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import { text, withKnobs } from '@storybook/addon-knobs'; // eslint-disable-line import/no-extraneous-dependencies
+import { Header } from '@textkernel/oneui';
+
+storiesOf('Header', module)
+    .addDecorator(withKnobs)
+    .add('Header', () => (
+        <Header
+            logoSrc={text('Image URL', 'jobfeed-logo.svg')}
```

Comment:
Maybe you can just link to https://www.jobfeed.nl/images/jobfeed-logo.svg so you don't have to add it as static resource ;) But doesn't matter that much

Reply:
ok